internal/command/machine/clone.go

```diff
@@ -77,6 +77,11 @@ func newClone() *cobra.Command {
 			Name:        "standby-for",
 			Description: "Comma separated list of machine ids to watch for. You can use '--standby-for=source' to create a standby for the cloned machine",
 		},
+		flag.Bool{
+			Name:        "require-unique-zone",
```

Comment:
can we name it `--volume-requires-unique-zone` ?  the amount of flags tend to grow on the `fly machines clone|run` commands

Reply:
Ah good call